Add a news post for Hecthor Doomhammers Tournament

obrakmann opened the pull request.

obrakmann commented:
Can someone please review this? It's gonna be a bit late to go up otherwise.

abcdefg30 commented on content/gamenews:
```diff
@@ -1,4 +1,4 @@
+2016-06-20 17-00-00:
+	Title: Red Alert 1vs1 Tournament coming up!
+	Author: Hecthor Doomhammer
+	Content: Our community member and well-known caster Hector Doomhammer will be hosting a tournament beginning on July 2nd.\n\nThe tournament will feature unique rules that will subject contestants to a new challenge each round to keep them out of their comfort zone and constantly on their toes.\n\nRunning over two weekends, a field of 16 challengers will fight to decide who will take the honor, g ...
```
`Hector` -> `Hecthor` I guess.

abcdefg30 commented:
Looks good to me otherwise. Can't test it though, so only :+0.5:

obrakmann commented:
Thanks, updated. Also added a screenshot so you can check it out without fighting nanoc.
![screenshot from 2016-06-21 22 41 30](https://cloud.githubusercontent.com/assets/4331210/16245534/9c758b32-3801-11e6-9478-b553a3ac1733.png)

pchote commented on content/news/doomhammer-tournament-announcement.md:
```diff
@@ -0,0 +1,20 @@
+
+Greetings ladies and gentlemen,
+
+our good friend Hecthor Doomhammer is eager to welcome you all to the first ever Hecthor Doomhammer Red Alert 1vs1 Tournament!
```
Capital O?

pchote replied:
Minor nit: it's generally good practice to avoid line-breaking in the middle of a title like Red Alert.

obrakmann replied:
I reworded the sentence that the split now sits between "Red Alert" and "1vs1"

pchote commented on content/news/doomhammer-tournament-announcement.yaml:
```diff
@@ -0,0 +1,7 @@
+---
+kind: article
+title: "Announcing Hecthor Doomhammer's Red Alert Tournament"
+author: "Hecthor Doomhammer"
```
It's a bit weird to have this "written" by Hecthor, referring to himself as his own good friend.

obrakmann replied:
Yup, changed

abcdefg30 commented:
> Thanks, updated. Also added a screenshot so you can check it out without fighting nanoc.

Nice thanks. Looks good. :) ✅

obrakmann commented on content/news/doomhammer-tournament-announcement.md:
```diff
@@ -0,0 +1,20 @@
+
+The complete tournament will be hosted and livestreamed from [Hecthor Doomhammer's Twitch channel](http://www.twitch.tv/pauljongejans). Joining him will be his fellow casters SoScared and FiveAces as well as well-known community figures Jazz_KCS and Ripsn as tournament officials.
+
+To register visit the [Tournament thread on Sleipnir's Forum](http://forum.openra.net/82/19594). You will also find all about the specific tournament rules in that thread. Registrations will close on June 26th, so hurry hup!
```
Oh wait, this link doesn't work when ihptru.net is down.

obrakmann replied:
Changed the links to not use the redirects.

pchote commented on content/gamenews:
```diff
@@ -1,4 +1,4 @@
-	Content: We are happy to announce the latest stable release of OpenRA today!\n\nThe big stand-out features are under the hood this time, but there are still lots of things to look forward to in this release: targetable husks on refineries, a restart button, lots of bug fixes and many things more.\n\nHead over to http://openra.net to read the full announcement, and visit http://changelog.openra.n ...
+2016-06-21 20-00-00:
+	Title: Red Alert 1vs1 Tournament coming up!
+	Author: Hecthor Doomhammer
```
Same issue here FYI

obrakmann replied:
derp. fixed as well
